### PR TITLE
Added sample python code for API usage

### PR DESCRIPTION
**Pull request by osintalex**
Hi!

I've been using the API within some Python programs so thought it could be useful to include some sample python code on how to use the API.

### CODE REVIEW

File: Code samples for API usage/sampleCode.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+# Create a class with attributes that relate to VideoIndexer credentials
+@dataclass
+class VideoIndexer:
+    subscription_key: str = "SUBSCRIPTION KEY GOES HERE"
+    account_id: str = "ACCOUNT ID GOES HERE"
+    location: str = "TRIAL"  # change this if you have a paid subscription tied to a specific location
+
+    @classmethod
+    def get_access_token(cls):
```
Comment (ika-microsoft) [inline Code samples for API usage/sampleCode.py:15]: Should be specific: get_account_access_token

File: Code samples for API usage/sampleCode.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        :param video_id: string, identifier for the video..
+        :param video_url: string, public url for the video.
+        :param access_token: string, required to use the API.
+        :return video_indexer_id: string, used to access video details once indexing complete.
+        """
+
+        # Set request headers and url
+        headers = {
+            "Content-Type": "multipart/form-data",
+            "Ocp-Apim-Subscription-Key": cls.subscription_key,
```
Comment (ika-microsoft) [inline Code samples for API usage/sampleCode.py:49]: You don't need to send the subscription key. You are authenticated using the access token

File: Code samples for API usage/sampleCode.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        """
+        Retrieves data on the video after analysis from the Video Indexer API.
+        :param video_id: string, unique identifier for the indexed video.
+        :param access_token: string, required to use the API.
+        :return video_indexer_json_output: JSON, analysed video data.
+        """
+
+        # Set request headers and url
+        headers = {
+            "Ocp-Apim-Subscription-Key": cls.subscription_key,
```
Comment (ika-microsoft) [inline Code samples for API usage/sampleCode.py:112]: Not required

**Review by ika-microsoft — APPROVED** [review]
Please fix comments. Other than that, well done

**Comment by osintalex** [comment]
Hey, thanks for the feedback! I think I've made the changes you wanted, let me know if you need anything else!

File: Code samples for API usage/sampleCode.py (hunk 3 above)
Comment (tshaiman) [reply Code samples for API usage/sampleCode.py]: @ika-microsoft  : is this PR ready for merge ?

**Comment by tshaiman** [comment]
closed by mistake

**Comment by tshaiman** [comment]
@osintalex  : can you please reopen this PR , it was closed by mistake

**Comment by osintalex** [comment]
Just had a look but a bit confused since it looks like there are classic/arm options since I last worked on it. Is the same python code usable for both?

**Pull request closed without merging**